include/README

```diff
@@ -0,0 +1,76 @@
+ODP specification
+=================
+
+ODP specification consists of several types of files, which together provide
+full list of types, values and functions that ODP implemention MUST provide.
+
+ODP API specification
+---------------------
+
+These are the files from `include/odp/api/spec` directory. They specify a set
+of functions that MUST be provided by ODP implementation. Doxygen comments
+inside these files document requirements for ABI interface provided by an
+implementation. An implementation MUST use these headers AS IS, without any
+modifications to be compatible with ODP specification.
+
+ODP ABI compatibility specification
+-----------------------------------
+
+These are the files from `include/odp/arch/@ARCH_ABI@/odp/api/abi/` directory.
+They specify a set of types and values that MUST be used AS IS without any
+modifications by an implementation if it supports and is compiled for
+ABI-compatibility mode.
+
+ODP default ABI headers
+-----------------------
+
+These are the files from `include/odp/api/abi-default` directory. They provide
+default specification for ODP types and values. ODP ABI compatibility files
+heavily depend on these headers. These files MUST NOT be changed by an
+implementation. If an implementation is compiled for ABI-compatibility mode,
+these headers should be used AS IS.
+
```

Comment:
.... They provide default specification for ODP types and values for ABI compatibility. CPU architecture specific ODP ABI compatibility files depend heavily ...

The last sentence can be left out. MUST NOT be changed == MUST be used AS IS.